src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -24,6 +25,12 @@ internal sealed partial class LocalRewriter : BoundTreeRewriterWithStackGuard
         private bool _sawAwaitInExceptionHandler;
         private readonly DiagnosticBag _diagnostics;
 
+        /// <summary>
+        /// A lazily created delegate cache container of <see cref="DelegateCacheContainerKind.MethodScopedGeneric"/>.
+        /// </summary>
+        private TypeOrMethodScopedDelegateCacheContainer _methodScopedDelegateCacheContainer;
```

Comment:
Please include "generic" in the name. For example, _methodScoped**Generic**DelegateCacheContainer